### DISCUSSION TITLE
Mails sent to intial superadmin rejected

### DISCUSSION THREAD

**Discussion by nicosqc** [modoboa/modoboa#2563] [Help] [answered]
The mails sent to admin(@maindomain.com) are getting rejected when coming from an external source. This effectively locked me out of accounts that use this email as a 2FA. I have tried adding a simpleuser and an alias with "admin@maindomain.com" and in both cases it is still the same result. Then I went into the settings of admin and checked the "Mail" checkbox, adding a mailbox to the admin account itself which I apparently cannot undo. I can send emails from this account but when I reply it gets rejected as follows:

```
This is the mail system at host myother.server.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<admin@maindomain.com>: host mail.maindomain.com[1.2.3.4]
    said: 550 5.1.1 <admin@maindomain.com>: Recipient address rejected:
    undeliverable address: host
    mail.maindomain.com[private/dovecot-lmtp] said: 550 5.1.1
    <admin@maindomain.com> User doesn't exist:
    admin@maindomain.com (in reply to RCPT TO command) (in reply to
    RCPT TO command)

```

I don't understand why they would get rejected. Everything else works, sending from another domain thats on the instance to the account works fine. Sending mails to other admin@secondarydomain.com also works fine. I just cant send mail to this specific account from the outside, which is quite problematic as I have set up things to use this mail address in the past.

**Comment by nicosqc** [comment, marked as answer]
Oh well. I created a new superadmin, logged into this one and deleted the old one. Recreated the alias and now it works as intended.

**Reply by tonioo** [reply]
The default admin does not have any mailbox, that's why you can't send him messages or access the webmail...

**Reply by nicosqc** [reply]
I did go into the settings of the initial admin and activated the mailbox. I could send mail from local users (on the same machine) to this admin, but mails from outside of the mailserver got rejected. Either way, its working fine right now with the new superadmin and admin@ as an alias to a regular mailbox
